sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusReceivedMessage.java

```diff
@@ -508,12 +523,12 @@ void setEnqueuedTime(OffsetDateTime enqueuedTime) {
     }
 
     /**
-     * Sets the subject for the message.
+     * Sets whether the message has been settled.
      *
-     * @param subject The subject to set.
+     * @param isSettled True if the message has been settled and false otherwise.
      */
-    void setSubject(String subject) {
-        amqpAnnotatedMessage.getProperties().setSubject(subject);
+    void setIsSettled(boolean isSettled) {
+        this.isSettled = isSettled;
```

Comment:
Do we need argument. ? Shouldn't this always set it to true

Reply:
Resolved